addons/main/functions/fnc_EHKilled.sqf

```diff
@@ -51,3 +73,5 @@ clearAllItemsFromBackpack _backpack;
 
 //remove the backpack from the dead unit
 [_unit] call FUNC(removeChestpack);
+
+_holder
```

Comment:
```suggestion

```
Remove return

Reply:
Returning `_holder` is needed in `fnc_actionLower` L29 `private _holder = [_unit] call FUNC(EHKilled);`  
After the holder is created, it needs to be attached to the rope.